portfolio/src/main/webapp/index.html

```diff
@@ -7,6 +7,7 @@
       integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
     <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
     <link rel="stylesheet" href="style.css">
+    <script defer src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBIHCvJyyPWTcZrste8DREkiKusDJMfJ4Y&callback=initMap" type="text/javascript"></script>
```

Comment:
According to the documentation, this line of code should be ```<script async defer src="https...``` but when I add ```async``` it causes a harmless error in the Javascript Console. Should I add ```async```, the map still works with the error.

Reply:
Here is the error:

![image](https://user-images.githubusercontent.com/53728084/84078727-9f3d3a80-a996-11ea-878d-d4c60d3e4593.png)

Reply:
Interesting, not positive why that's happening only when you include `async` but you can read a bit more about it and `defer` [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script). I tend to use `defer` for most scripts so I don't block parsing of the HTML and can still control the execution order. See if putting that on both works.

Reply:
When I add defer to both it gave this error instead: 

![image](https://user-images.githubusercontent.com/53728084/84091969-46c86600-a9b3-11ea-9526-1debf6923bed.png)